hack/preview.sh

```diff
@@ -33,13 +33,16 @@ if [ -n "$PIPELINE_SERVICE_IDENTITY_HASH" ]; then
   IDENTITY_HASHES="pipeline-service:$PIPELINE_SERVICE_IDENTITY_HASH"
 else
   KUBECONFIG=${KCP_KUBECONFIG} kubectl ws ${ROOT_WORKSPACE}
-  if KUBECONFIG=${KCP_KUBECONFIG} kubectl ws pipeline-service; then
+  if KUBECONFIG=${KCP_KUBECONFIG} kubectl ws redhat-pipeline-service-compute; then
     IDENTITY_HASH=$(oc get apiexport kubernetes --kubeconfig ${KCP_KUBECONFIG} -o jsonpath='{.status.identityHash}')
     IDENTITY_HASHES="pipeline-service:$IDENTITY_HASH"
+  elif [ "$PIPELINE_SERVICE_LOCAL_DEPLOY" == "true" ]; then
+    KUBECONFIG={CLUSTER_KUBECONFIG} ./hack/install-pipeline-service.sh
```

Comment:
but in this case, we need to wait until the APIExport is created and then take the identity hash right?
something like this:
```suggestion
    KUBECONFIG={CLUSTER_KUBECONFIG} ./hack/install-pipeline-service.sh
    IDENTITY_HASH=$(oc get apiexport kubernetes --kubeconfig ${KCP_KUBECONFIG} -o jsonpath='{.status.identityHash}')
    IDENTITY_HASHES="pipeline-service:$IDENTITY_HASH"
```

Reply:
updated, thanks (also fixed install-pipeline-service)